sklearn/utils/testing.py

```diff
@@ -800,6 +802,16 @@ def _get_args(function, varargs=False):
         return args
 
 
+def _get_parent_args(Estimator):
+    """Helper to get list of parents properties"""
```

Comment:
parents -> parents'.

I don't think "properties" is correct. You've got a mix of constructor parameter names and class attribute names. I don't think `_get_parent_args` is a sufficient name for this either, given that `_get_args` does not return attributes.